### ISSUE TITLE
Invert helper

### ISSUE THREAD

**Issue by Morganjackson** [styled-components/polished#152]
I have a `setTextColor()` function that returns black or white based on the background color, and I was wondering if polished could do this?

```javascript
export function setTextColor(color) {
  const rgb = convertHexToRgb(color)
  const gamma = 2.2
  const luminosity =
    0.2126 * Math.pow(splitRgb(rgb).r, gamma) +
    0.7152 * Math.pow(splitRgb(rgb).g, gamma) +
    0.0722 * Math.pow(splitRgb(rgb).b, gamma)
  if (luminosity < 0.6) return '#FFFFFF'
  return '#000000'
}
```

I found [invert()](https://github.com/styled-components/polished/blob/94db3ed0a0cd156a2bf745832c8d219786984762/src/color/invert.js#L29-L38) which works in some cases but I find you usually just want black or white for things like buttons or navigations.

**Comment by bhough** [comment]
@Morganjackson Good news, it is already being discussed as part of #148.

**Comment by bhough** [comment]
@Morganjackson we have merged this in for v2 #185
